### ISSUE TITLE
Fine-tuning quesiton

### ISSUE THREAD

**Issue by madalinmarius** [unit8co/darts#1899]
In the current version, it's fine-tuning possible? I saw an older post about that and I wonder if it's a way in the current version to achieve that.

**Comment by JanFidor** [comment]
Hi @madalinmarius ! It should be supported by models which can be trained on multiple TimeSeries (there's a column with info which models support that in the Readme.md). Here's also a great notebook if you're looking for an example how to go about coding that up https://unit8co.github.io/darts/examples/14-transfer-learning.html?highlight=transfer !

**Comment by madalinmarius** [comment]
Hi @JanFidor ! So far I saw, they do inference on another dataset without retraining the model on that dataset too.

I'm looking for a way to train a model "x" on a bigger dataset and then retrain it on a smaller one. In the beginning, I thought that was possible with the framework, without writing everything from scratch.

**Comment by JanFidor** [comment]
Ah gotcha, I can't find anything in docs about that, but I browsed the code for the base models and I think that Global models shouldn't throw errors if you call fit multiple times (separately for each dataset). The one problem is that learning rate (lr_rate) is defined for the whole model, not just training and I'm not sure if it's possible to cleanly change it

**Comment by dennisbader** [comment]
With the new darts release coming at the end of July, our TorchForecastingModels will properly support fine tuning by loading the weights (with [`load_weights`](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.tft_model.html#darts.models.forecasting.tft_model.TFTModel.load_weights) or [`load_weights_from_checkpoint`](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.tft_model.html#darts.models.forecasting.tft_model.TFTModel.load_weights_from_checkpoint)) of a pre-trained model into a new model instance with different optimizer, learning rate (scheduling), etc.

The other global models such as RegressionModels will just retrain (overwrite) the model on the new data as if it had not been fit before.

**Comment by madalinmarius** [comment]
That's amazing. Thank you @dennisbader and @JanFidor for all the help. This update will be really good :)
